### ISSUE TITLE
[BUG] Everything is frozen after a change of skin (Bordeaux)

### ISSUE THREAD

**Issue by Myllaume** [Zettlr/Zettlr#970]
## Description

At the first installation, everything was working. I changed the skin: Bordeaux. The software doesn't respond to commands anymore, despite uninstalling and reinstalling. I switch to 1.6 and changed the skin (the software was logically naked): Berlin and everything works again.

## Reproducing

So sorry, I couldn't reproduce the bug: after repeated uninstallation and reboot, the software in 1.7 doesn't respond to any command.

## Expected behaviour

When I restarted the software after the first skin change (in 1.7, to Bordeaux) :

![X6D17_pO jpg large](https://user-images.githubusercontent.com/37468701/86459038-7f472f80-bd26-11ea-9611-b6e3e9a8ed1d.jpg)

## Platform
 - OS and version: Windows 10.0.18363
 - Zettlr Version: 1.7.0
 - Screen Resolution: 1920x1080

Thank you very much for your great work for a great tool !

**Comment by nathanlesage** [comment]
This almost looks as if the Bordeaux-theme file wasn't correctly loaded … mhh. I'll try to reproduce here locally! Curious that this has never happened during the betas …

**Comment by emarshphd** [comment]
I have the same problem on MacOS. I was using the beta before and using Bordeaux and it wasn't a problem. Before this issue, I also had the problem mentioned elsewhere with editing links.

**Comment by paulasimoes** [comment]
I've experienced the same exact problem in Kubuntu 20.04, after changing the theme to Bordeaux. In one of attempts to open Zettlr, after a few minutes it became responsive, so I went to preferences, changed the theme to the green one. It seemed ok, but soon it became not responsive again. After 3 or 4 minutes, it started working again normally. During the problems, laptop's fans were working noisily - zettlr was consuming 13% of CPU. I closed Zettlr and started it again, to replicate the problem: when opening Zettlr it freezed for a couple of minutes, 13%CPU,  after that it seemed ok, then when moving through notes, it freezed again. Then after another couple of minutes, it changed to another note I clicked before (when feezed), then freezed again.
